doc/tutorials/acrn-secure-boot-with-grub.rst

```diff
@@ -0,0 +1,177 @@
+.. _how-to-enable-acrn-secure-boot-with-grub:
+
+Enable ACRN Secure Boot with GRUB
+#################################
+
+This document shows how to enable ACRN secure boot with GRUB including:
+
+  -  ACRN Secure Boot Sequence
+  -  Generate GPG Key
+  -  Setup Standalone GRUB EFI Binary
+  -  UEFI Secure Boot
+
+  Validation Environment:
+	Hardware Platform: INTEL WHL
+	ACRN Scenario: Industry
+	Service VM: Ubuntu 18.04
+	GRUB: 2.02
+
+Note: Please make sure you have alternative ways to restore GRUB when you got stuck.
+
+ACRN Secure Boot Sequence
+*************************
+
+  ACRN can be booted by Multiboot compatible bootloader, following diagram illustrates the boot sequence of ACRN with GRUB:
+
+     .. image:: images/acrn-secureboot-flow.png
+        :align: center
+        :width: 600px
+
+
+  Refer to :ref:`using_grub` for details to enable GRUB on ACRN.
+
+  From point view of secureboot:
+	- UEFI firmware verifies Shim/GRUB
+	- GRUB verifies ACRN, Service VM kernel and Prelaunch VM kernel
+	- SOS kernel verifies DM and UOS OVMF thru ACRN DM
+	- UOS OVMF starts the guest side verified boot process
```

Comment:
```suggestion
	- User VM OVMF starts the guest side verified boot process
```

I'm wondering if it wouldn't be better to talk about the "virtual bootloader" instead of "User VM OVMF". @dbkinder , thoughts?